src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
+type Color = 'red' | 'green' | 'blue';
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Color,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
```

Comment:
It is bad practice to make fields public when unnecessary because it violates the encapsulation OOP principle. Even accessing shape and color properties from outside the class is not so good, would be nice to have a method for printing such information, but task requires that